orm6/mapper/orm-coordination-outbox-polling/ant-src-changes.patch

```diff
@@ -1,5 +1,38 @@
+diff --git a/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java b/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java
```

Comment:
I suppose you used the scripts to update the patches?

Can you please push the corresponding commits to the `wip/main/dependency-update/orm6-in-main-code` branch?